queue/queue.py

```diff
@@ -10,30 +10,37 @@ def __init__(self):
         """
         Initialize the Queue with an empty LinkedList
         """
-        pass
+        super().__init__()
+        self.items = LinkedList()
 
     def is_empty(self):
         """
         :returns: True if the LinkedList is empty and False if it isn't
         """
-        pass
+        if not self.items.head:
+            return True
+        return False
 
     def enqueue(self, value):
         """
         Adds the value to the back of the LinkedList
         :param value: the value to be added
         """
-        pass
+        self.items.add(value)
 
     def dequeue(self):
         """
         Removes the value at the head
         :return: the value of the item at the head
         """
-        pass
+        current_node = self.items.head
+        if not self.items.head:
+            return self.items.head.value
```

Comment:
Why are you returning the head's value when there's no head? I think you should be returning `None` no? cc @jirevwe

Reply:
True

Reply:
Yep yep.